client/java/src/main/java/com/linecorp/centraldogma/internal/client/ReplicationLagTolerantCentralDogma.java

```diff
@@ -333,7 +492,17 @@ public ReplicationLagTolerantCentralDogma(ScheduledExecutorService executor, Cen
             Function<Revision, CompletableFuture<T>> taskRunner) {
         return normalizeRevision(projectName, repositoryName, revision)
                 .thenCompose(normRev -> executeWithRetries(
-                        () -> taskRunner.apply(normRev),
+                        new Supplier<CompletableFuture<T>>() {
+                            @Override
+                            public CompletableFuture<T> get() {
+                                return taskRunner.apply(normRev);
+                            }
+
+                            @Override
+                            public String toString() {
+                                return taskRunner + " with " + normRev;
```

Comment:
nit: `taskRunner with (normRev)` like below?

Reply:
I wrapped with `()` because it's a range. I guess I need to wrap with square brackets though, because it's inclusive at both ends.